content/doc/book/managing/system-configuration.adoc

```diff
@@ -13,3 +13,114 @@ endif::[]
+Configuration layers might include Jenkins master configuration, 
+link:https://www.jenkins.io/doc/book/using/using-agents/[agents configuration],
+link:https://www.jenkins.io/doc/book/system-administration/security/[security configuration].
+
+Configuration pages are frequently created and expanded by plugins, maybe already exists a plugin to resolve your needs.
+Consider visiting the link:https://plugins.jenkins.io/[Jenkins plugins page] to check if any of them could be useful to you.
+
+Configuration techniques might include:
+
+== Configuration from the UI
+
+The UI configuration is available under the Manage Jenkins menu item. 
+There exists 3 main sections: System Configuration, Security, Status Information.
+each one has their own options:
+
+=== System Configuration options
+
+==== Configure System
+
+The Configure System has native options and can receive new ones from plugins.
+The main Jenkins installation can be modified by plugins, such as System message, data and time format, Jenkins locations, SMTP Server, and others.
+
+==== Global Tool Configuration
+
+The Global Tool Configuration keeps together all information about the tool which Jenkins uses to run each task.
+Maven, JDK, Git, Gradle, and Ant are tools present at default installations.
+
+==== Manage Plugins
+
+The Manage Plugins is the place where is possible to add, update, remove, or know more about plugins.
+It is possible to request plugins from the web or uploads your own plugin by upload field.
+Unter the advanced section it is possible to set the proxy configurations or changes the Update site URL.
+
+==== Manage Nodes and Clouds
+
+The Manage Nodes and Clouds menu is responsible for managing all information about all Jenkins agents.
+from the initial page, you can add, remove, enable, or not.
+Please visit the link:https://www.jenkins.io/doc/book/using/using-agents/[agents configuration page] to know more about it.
+
+=== System Security options
+
+==== Configure Global Security 
+
+The Configure Global Security has all basic configuration to general security such as Authentication methods where it is possible
+set Jenkins to use their own database or LDAP use, or set the strategy to authorization and others. 
+The link:https://www.jenkins.io/doc/book/system-administration/security/[Security page] has all informations about the Jenkins  security.
+
+==== Manager Credentials
+
+The Manager Credentials is the right place to store all your credentials access to several services like ssh key, git, svn, and others.
+
+==== Configure Credential Providers
+
+==== Manager Users
+
+The Manager Users menu is the place where it is possible to access information about users in the Jenkins’ own user database.
+Visit the link:https://www.jenkins.io/doc/book/system-administration/authenticating-scripted-clients/[Script Authentication Clients page] to learn how to auth clients using the API.
+
+=== Status Information
```

Comment:
IMHO it rather belongs to a separate page about troubleshooting Jenkins. Could be left here for now